<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorScopeConfigWidget</class>
 <widget class="QWidget" name="MotorScopeConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="dataChannelLabel">
     <property name="text">
      <string>Data Channel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="dataChannelSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="triggerChannelLabel">
     <property name="text">
      <string>Trigger Channel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="triggerChannelSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="verticalRangeLabel">
     <property name="text">
      <string>Vertical Range</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="verticalRangeDoubleSpinBox">
     <property name="suffix">
      <string> V</string>
     </property>
     <property name="maximum">
      <double>20.000000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="recordLengthLabel">
     <property name="text">
      <string>Record Length</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="recordLengthSpinBox">
     <property name="maximum">
      <number>99999999</number>
     </property>
     <property name="value">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="sampleRateLabel">
     <property name="text">
      <string>Sample Rate</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="sampleRateDoubleSpinBox">
     <property name="suffix">
      <string> ns/Sa</string>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="value">
      <double>500.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="triggerDirectionLabel">
     <property name="text">
      <string>Trigger Direction</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="triggerDirectionComboBox">
     <item>
      <property name="text">
       <string>Rising</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Falling</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
